tools/ds-identify

```diff
@@ -179,10 +179,30 @@ debug() {
     echo "$@" 1>&3
 }
 
+map_sysfs_to_dmi() {
+    case "$@" in
```

Comment:
this should just look at $1, not $@.  You can error if you'd like if there are mroe than 1 arg.  but "$@" here is just confusing.